api/roleAccess/index.ts

```diff
@@ -6,9 +6,57 @@ import { getConfigValue } from '../configuration';
 import { SERVICES_ROLE_ASSIGNMENT_API_PATH } from '../configuration/references';
 import { EnhancedRequest } from '../lib/models';
 import { toRoleAssignmentBody } from './dtos/to-role-assignment-dto';
+import { CaseRoleRequestPayload } from './models/caseRoleRequestPayload';
+import { setHeaders } from '../lib/proxy';
+import { RoleAssignment } from '../user/interfaces/roleAssignment';
+import { CaseRole } from '../workAllocation2/interfaces/caseRole';
+import { release2ContentType } from './models/release2ContentType';
+import { http } from '../lib/http';
+import { getEmail, getUserName, mapRoleCategory } from './exclusionService';
 
 const baseRoleAccessUrl = getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH);
 
+export async function getRolesByCaseId(req: EnhancedRequest, res: Response, next: NextFunction): Promise<Response> {
+  const requestPayload = getLegalAndJudicialRequestPayload(req.body.caseId, req.body.jurisdiction, req.body.caseType);
+  const basePath = getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH);
+  const fullPath = `${basePath}/am/role-assignments/query`;
+  const headers = setHeaders(req, release2ContentType);
+  try {
+    console.log(JSON.stringify(requestPayload))
+    const response: AxiosResponse = await http.post(fullPath, requestPayload, {headers});
+    console.log(response.data);
```

Comment:
Minor comment - Are the console.logs necessary?

Reply:
Removed it